addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionPluginUnitTest.java

```diff
@@ -56,9 +56,9 @@ protected int getRecommendMaxNumberMessagesPerParam(AttackStrength strength) {
             default:
                 return recommendMax + 20;
             case HIGH:
-                return recommendMax + 22;
+                return recommendMax + 27;
             case INSANE:
-                return recommendMax;
+                return recommendMax + 14;
```

Comment:
Not Sure about this test. Is this test is just for the count of payloads ? if not please specify.

Reply:
thanks for the reply.